packages/game-bridge/src/index.ts

```diff
@@ -509,6 +510,17 @@ window.callFunction = async (jsonData: string) => {
         });
         break;
       }
+      case trackFunction: {
+        const request = JSON.parse(data);
+        const properties = JSON.parse(request.properties);
```

Comment:
what damange might be if any error happen during parse, or callbackToGame function call during this Track case?

Reply:
It goes to the catch block below, and the error propagates back to the game. The Unity SDK however will ignore any errors thrown by this function.